hazelcast/src/main/java/com/hazelcast/spi/impl/operationexecutor/ResponsePacketHandler.java

```diff
@@ -0,0 +1,15 @@
+package com.hazelcast.spi.impl.operationexecutor;
+
+import com.hazelcast.nio.Packet;
+import com.hazelcast.spi.impl.Response;
+
+/**
+ * The {@link com.hazelcast.spi.impl.operationexecutor.ResponsePacketHandler} is responsible for handling
```

Comment:
so if I pass a Response to this handler, what behaviour can I expect? will it send the response to a caller? what is the caller can't be reached? 

besically a description of expected behaviour for future generations :)